qiskit_ibm_runtime/ibm_runtime_service.py

```diff
@@ -658,7 +660,11 @@ def pprint_programs(
                 print(f"  Description: {prog.description}")
 
     def programs(
-        self, refresh: bool = False, limit: int = 20, skip: int = 0
+        self,
+        refresh: bool = False,
+        name: Optional[str] = "",
+        limit: int = 20,
+        skip: int = 0,
```

Comment:
This way, this change is not backward-compatible. Users may rely on the order of the positional arguments (limit, skip). An easy way to fix that is to add the optional name parameter at the end.